v0/destinations/ga4/utils.js

```diff
@@ -357,8 +358,39 @@ const getGA4CustomParameters = (message, keys, exclusionFields, payload) => {
 const responseHandler = (destinationResponse, dest) => {
   const message = `[GA4 Response Handler] - Request Processed Successfully`;
   let { status } = destinationResponse;
+  const { response } = destinationResponse;
   if (status === 204) {
     status = 200;
+    // for GA4 debug validation endpoint, status is always 200
+  } else if (status === 200 && isDefinedAndNotNull(response)) {
```

Comment:
```suggestion
  } else if (status === 200 && isDefinedAndNotNull(response) && isDefined(response.validationMessages)) {
```
This way we can know that this particular status 200 is coming from debug mode only.

Reply:
the 2nd if validation is not be required while implementing this logic

Reply:
Updated.